Automattic/simplenote-macos
# About: Adds Get Help row

### Details:
In this PR we're adding a new `Get Help` row!

@guarani Sir!! can I trouble you with this one?
cc @SylvesterWilmott for UI reasons!!

### Test
1. Launch Simplenote
2. Open the **About Window**

- [ ] Verify the mouse switches to a pointing cursor as you hover the different entries
- [ ] Verify every clickable item opens a Web onClick

### Release
`RELEASE-NOTES.txt` was updated in 0418542 with:

> New `Get Help` action in Simplenote's About window

### Screenshots
<img width="300" alt="About" src="https://user-images.githubusercontent.com/1195260/95222562-00530e80-07cf-11eb-9fbf-a3eacf208a79.png">
